R/plotPrevalence.R

```diff
@@ -0,0 +1,322 @@
+#' Plot prevalence information
+#' 
```

Comment:
I tested this function:
```
plotTaxaPrevalence(GlobalPatterns, rank = "Family",
                   detections = c(0, 0.0001, 0.001, 0.01, 0.1, 1))
plotTaxaPrevalence(GlobalPatterns, rank = "Genus",
                   detections = c(0, 0.0001, 0.001, 0.01, 0.1, 1))
plotTaxaPrevalence(GlobalPatterns, rank = "Species",
                   detections = c(0, 0.0001, 0.001, 0.01, 0.1, 1))
```
From family to species the plot is 


![image](https://user-images.githubusercontent.com/16476233/105197643-1fd33480-5b3d-11eb-83b6-b60ab1572c5c.png)

With `SprockettTHData`  species and family level shows similar plot as above. 
E.g. below Genus plot that works, but we can have a min.prevalence option, like in the `microbiome` package to avoid over-crowding with large regions of blue shaded taxa in the end of the figure. 
![image](https://user-images.githubusercontent.com/16476233/105198007-848e8f00-5b3d-11eb-8f29-2cc26bceeb60.png)

Reply:
Yes, this would be good. Also: is the color scale legend now in [0, 1] or in [0, 100] - If the unit (%) is given then I think also the numbers should be in [0, 100]. @tvborman could check this?